lib/dependency-manager-adapters/workspace.js

```diff
@@ -28,7 +28,11 @@ module.exports = CoreObject.extend({
     }
 
     let packageJSON = JSON.parse(fs.readFileSync(this.packageJSON));
-    let workspaceGlobs = packageJSON.workspaces;
+    let workspaceGlobs = packageJSON.workspaces
+
+    if (workspaceGlobs && workspaceGlobs.packages) {
+      workspaceGlobs = workspaceGlobs.packages
+    }
```

Comment:
Can you tweak this to be specific about what we expect? I'm thinking of something like:

```js
let workspaceGlobs = Array.isArray(packageJSON.workspaces) ? packageJSON.workspaces : packageJSON.workspaces.packages;
```

Reply:
Sure! Ended up with

```js
if (Array.isArray(packageJSON.workspaces)) {
  workspaceGlobs = packageJSON.workspaces
}

if (packageJSON.workspaces && Array.isArray(packageJSON.workspaces.packages)) {
  workspaceGlobs = packageJSON.workspaces.packages
}
```

so it's specific about each case